### PR TITLE
add-disable-macro

### PR DESCRIPTION
**Pull request by Asbjoedt**

### CODE REVIEW

File: v1.0/Specification.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ The following table gives guidelines for how to comply with the requirements in
...
+| ODS_7 | Macros contain programming code that can execute and manipulate data in the spreadsheet and the file system. These may present security risks, and it is therefore prudent to remove or disable any macros. Macros add interaction to a spreadsheet and can change data.<br><br>Removing macros will prevent any data from changing, but the macro itself will be lost if not documented other places.<br><br>Another option is to set up your spreadsheet rendering application to disable any macros. This will prevent any data from changing, it will preserve the interactivity of the macro and the macro may be reused by a user to support original reuse scenarios, but the same interactivity may also lead to change of data in unintended and non-transparent ways, if the user is not careful. |
```
Comment (carlwilson) [inline v1.0/Specification.md:186]: I feel that the requirement is becoming way too long here in an attempt to present the rationale. The hints as to what an archivist might do about macros don't belong in the requirement.

Comment (Asbjoedt) [reply v1.0/Specification.md:186]: It's not the requirement/policy, it is the guideline section.

Comment (carlwilson) [reply v1.0/Specification.md:186]: I'm going to approve it for now, but the text is too long and rambling. I will catch it in a full document review.

**Review by carlwilson — APPROVED** [review]
I'm going to approve it for now, but the text is too long and rambling. I will catch it in a full document review.

**Pull request merged**